phpstan.neon

```diff
@@ -16,6 +16,8 @@ parameters:
     ignoreErrors:
         # The class was added in PHP 8.1
         - '/^Attribute class ReturnTypeWillChange does not exist.$/'
-
         # https://github.com/doctrine/collections/pull/282
         - '/Variable \$offset in isset\(\) always exists and is not nullable\./'
+        # method removed in doctrine/dbal3.x
+        - '/^Call to an undefined method Doctrine\\DBAL\\Platforms\\AbstractPlatform::getGuidExpression\(\).$/'
```

Comment:
This is only useful when using DBAL 3, and thus makes the DBAL 2 phpstan job fail :(
Maybe we should have a separate file for DBAL 3? Like `phpstan-dbal3.neon`?